src/test/java/teammates/test/pageobjects/FeedbackSubmitPage.java

```diff
@@ -193,4 +193,20 @@ public void waitForOtherOptionTextToBeClickable(int qnNumber, int responseNumber
         waitForElementToBeClickable(element);
     }
 
+    public void verifyVisibilityAndCloseMoreInfoAboutEqualShareModal() {
+        WebElement moreInfoAboutEqualShareModalLink = browser.driver.findElement(By.id("more-info-equal-share-modal-link"));
+        click(moreInfoAboutEqualShareModalLink);
+        WebElement moreInfoAboutEqualShareModal = browser.driver.findElement(By.id("more-info-equal-share-modal"));
+        waitForElementVisibility(moreInfoAboutEqualShareModal);
+        closeMoreInfoAboutEqualShareModal();
+    }
+
+    public void closeMoreInfoAboutEqualShareModal() {
+        WebElement closeButton = browser.driver.findElement(By
+                .xpath("//div[@id='more-info-equal-share-modal']//button[@class='close']"));
```

Comment:
These line wrappings are more readable (i.e. prefer line break at higher levels)
```java
WebElement closeButton = browser.driver.findElement(
        By.xpath("//div[@id='more-info-equal-share-modal']//button[@class='close']"));
```
```java
WebElement closeButton = browser.driver
        .findElement(By.xpath("//div[@id='more-info-equal-share-modal']//button[@class='close']"));
```
OR extract a local variable
```java
String closeButtonPath = "//div[@id='more-info-equal-share-modal']//button[@class='close']";
WebElement closeButton = browser.driver. findElement(By.xpath(closeButtonPath));
```
Now that I look at the 3rd option, I feel like it should be extracted into a utility method `getElementFromPath(String):WebElement`. May be it already exists somewhere, may be in the parent class?